contrib/mesos/pkg/scheduler/podtask/pod_task.go

```diff
@@ -271,10 +293,6 @@ func RecoverFrom(pod api.Pod) (*T, bool, error) {
 			offerId = v
 		case annotation.TaskIdKey:
 			t.ID = v
-		case annotation.ExecutorIdKey:
```

Comment:
Ah, here's why you removed the `annotation.ExecutorIdKey`, I guess this code fragment has been moved to where?